<launch>
	<node pkg="mini_robot" type="imu_publisher.py" name="imu_node" output="screen">
	</node>
</launch>
